### PR TITLE
Suggest tweaks to meltdown advisory

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: docs/advisories/meltdown-kernel-update.md [hunk 1]
```diff
@@ -1,40 +1,35 @@
-## Meltdown and Spectre Advisory
+## Kernel Update required for "Meltdown" issue
```
Comment (chrislovecnm) [inline docs/advisories/meltdown-kernel-update.md]: This addresses one of the Spectre issues as well.

File: docs/advisories/meltdown-kernel-update.md [hunk 2]
```diff
@@ -43,7 +38,8 @@ dmesg -H | grep 'page tables isolation'
 
 ## Impacted Maintained Component(s)
 
-- kops maintained AMI
+* Patches were released for the linux kernel 2018-01-05.  All images prior to this date likely need updates.
```
Comment (chrislovecnm) [inline docs/advisories/meltdown-kernel-update.md]: This is good information, but it does not match the heading.

File: docs/advisories/meltdown-kernel-update.md [hunk 3]
```diff
@@ -56,39 +52,46 @@ For the kops-maintained AMIs, the following AMIs contain an updated kernel:
...
 For all examples please replace `$CLUSTER` with the appropriate kops cluster
 name.
 
-#### Determine which instance groups exist
+#### List instance groups
 
 `kops get ig --name $CLUSTER`
 
-#### Edit the kops instance groups 
+#### Replace the image for each instance group
```
Comment (chrislovecnm) [inline docs/advisories/meltdown-kernel-update.md]: They are not replacing.  Maybe the wrong verb?  They are modifying the cluster???

Comment (chrislovecnm) [reply docs/advisories/meltdown-kernel-update.md]: nit pick btw

File: docs/advisories/README.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ## kops Advisories
 
-- [Meltdown and Spectre](meltdown-spectre.md)
+- [Meltdown Kernel Update Required](meltdown-kernel-update.md)
```
Comment (chrislovecnm) [inline docs/advisories/README.md]: I am confused, as I am reading that one spectre cve and one meltdown cve was fixed.

From https://googleprojectzero.blogspot.co.uk/2018/01/reading-privileged-memory-with-side.html

So far, there are three known variants of the issue:

Variant 1: bounds check bypass (CVE-2017-5753)
Variant 2: branch target injection (CVE-2017-5715)
Variant 3: rogue data cache load (CVE-2017-5754)

Before the issues described here were publicly disclosed, Daniel Gruss, Moritz Lipp, Yuval Yarom, Paul Kocher, Daniel Genkin, Michael Schwarz, Mike Hamburg, Stefan Mangard, Thomas Prescher and Werner Haas also reported them; their [writeups/blogposts/paper drafts] are at:

Spectre (variants 1 and 2)
Meltdown (variant 3)

Comment (justinsb) [reply docs/advisories/README.md]: Grrr... I think you're right.  I'll tweak - I'm just calling to call them spectre/meltdown.  The separate CVEs are just muddying the waters IMO

File: docs/advisories/meltdown-kernel-update.md (hunk 2 above)
Comment (justinsb) [reply docs/advisories/meltdown-kernel-update.md]: Not sure what you mean

File: docs/advisories/meltdown-kernel-update.md (hunk 3 above)
Comment (justinsb) [reply docs/advisories/meltdown-kernel-update.md]: Changing to `Update the image for each instance group`

**Pull request merged**